nemo/collections/asr/data/audio_to_text.py

```diff
@@ -266,6 +403,9 @@ def __init__(
         if type(manifest_filepath) == str:
             manifest_filepath = manifest_filepath.split(",")
 
+        # If necessary, cache manifests and audio from object store
+        cache_store_manifests(manifest_filepaths=manifest_filepath, cache_audio=True)
```

Comment:
If it was a class method you could call self.manifest_processor.cache_store_manifest() before below instantiation of ASRManifestProcessor. Just a thought, ignore if its extra work.